### ISSUE TITLE
Bug: Fatal application error

### ISSUE THREAD

**Issue by duckysuck** [project-error/npwd#916]
### Development Issue

Unsure

### Phone Version

1.71

### Issue Description

When opening PEFCL or crypto apps three errors appear
Fatal application error
The application has encountered an error it was unable to recover for

At the bottom in red | Cannot read properties of undefined (reading 'location')

### Issue Reproduction

Upgrade to latest version and install PEFCL or crypto.
open either app and witness error

**Comment by duckysuck** [comment]
If you need more info or testing let me know. Happy to help.

**Comment by itschip** [comment]
I'll try to reproduce this the coming week.

**Comment by georgino-ccrp** [comment]
hello, i have this issue but i cant solve it.. i have it on pefcl / crypto /services

**Comment by duckysuck** [comment]
> 

yes thats what I've said in description. Give Chip a chance to look at it. Quite a few others have the bug.

**Comment by georgino-ccrp** [comment]
yea... chip the man who will save us :D

**Comment by JarieTimmer** [comment]
I am having the same problem here, im using the latest releases for everything (pefcl, esx, nwpd).

**Comment by georgino-ccrp** [comment]
so, how does it look ? :D

**Comment by itschip** [comment]
For now, I would just use 1.7.0 until it is fixed.

**Comment by zhivotnoya** [comment]
I'm currently using 1.7.0 of npwd and still getting those errors.  I'm going thru the installations and configs with a fine tooth comb. However, what should be the starting order of qb-npwd, npwd, pefcl, and qb-pefcl?  Haven't found any clear documentation on this yet.

**Comment by JohnAllington** [comment]
We got the same issue.. is there any way to solve it? we are using latest versions of npwd + pefcl.. thanks for help^^

**Comment by jfrader** [comment]
https://github.com/search?q=repo%3Aproject-error%2Fnpwd%20%22.location%22&type=code

**Comment by jellyton69** [comment]
This is still an issue, not sure what's causing it I've been working on it for a couple of days now with no lock.
